evennia/web/website/templates/website/registration/password_reset_email.html

```diff
@@ -0,0 +1,15 @@
+{% autoescape off %}
+To initiate the password reset process for your {{ user.get_username }} {{ site_name }} account,
+click the link below:
+
+{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
```

Comment:
Currently, domain is set to "example.com" per the default configuration of what appears to be the Django Sites framework. This should probably be resolved during `evennia --init` but is beyond me to fix-- it also impacts the reverse "view on site" link in the Django admin (if get_absolute_url is implemented on the object in question).